### DISCUSSION TITLE
Priority Implementations

### DISCUSSION THREAD

**Discussion by gerdm** [gerdm/prml#1] [General]
Hello!
I hope everyone who is using this repository is finding it useful. I certainly am! I tend to come back to this repo whenever I need to recall an idea. Nevertheless, the aim of this project is not only to help myself, but all those who use this repo as well. Therefore, I've opened this discussion to know **what would you like to see implemented / fixed / commented.**

All ideas are welcome.

**Comment by alessandro-gentilini** [comment]
@gerdm Hello! Thank you for your sharing, regarding https://github.com/gerdm/prml/blob/master/chapter03/Bayesian-linear-regression.ipynb I would like to see implemented the case where \beta (or \sigma in your notebook) is unknown.
All the best,
Alessandro

**Reply by gerdm** [reply]
Hi @alessandro-gentilini! Thanks for the suggestion.
Please, feel free to create a PR if you'd like to work on this. Wikipedia has a very decent [article](https://en.wikipedia.org/wiki/Bayesian_linear_regression) on how to do this.
